cloudify_kubernetes/tests/test_tasks.py

```diff
@@ -710,7 +710,7 @@ def test_resource_delete(self):
                 with patch(
                         'cloudify_kubernetes.tasks._do_resource_read',
                         MagicMock()):
-                    with self.assertRaises(OperationRetry):
+                    with self.assertRaises(NonRecoverableError):
```

Comment:
Why?

Reply:
because I did update the following section and the raised exception should not be OperationRetry  

Check this https://github.com/cloudify-incubator/cloudify-kubernetes-plugin/pull/84/files/8e21140d2d95b647136b0000db745cb78b2be9ff#diff-fe79c324414532eb6335f4665e78b53bR143